### PR TITLE
Update /internet-of-things/appstore

### LINKED ISSUE #5994: Update/rebuild /internet-of-things/app-store page
Per the [copy doc](https://docs.google.com/document/d/1Wj-aq6qPnNMjgMY3yyBeJRjbBVVCRBq6s00JkR6-d_A/edit#)

* we should ask @anasereijo if she wants to advise before we update or after
* we need to move this copy doc over the old one, perhaps just copy and paste the new over the old to keep the history
* ignor the form in the doc, use the existing modal

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Updated /internet-of-things/appstore according to [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5994#issuecomment-546304958) and [copy doc](https://docs.google.com/document/d/1Wj-aq6qPnNMjgMY3yyBeJRjbBVVCRBq6s00JkR6-d_A/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/internet-of-things/appstore
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/5994#issuecomment-546304958) and [copy doc](https://docs.google.com/document/d/1Wj-aq6qPnNMjgMY3yyBeJRjbBVVCRBq6s00JkR6-d_A/edit#)

## Issue / Card

Fixes #5994

### CODE REVIEW

File: templates/internet-of-things/appstore.html [hunk 1]
```diff
@@ -195,52 +90,209 @@ <h2 class="p-muted-heading u-align--center">Used by</h2>
...
-      </p>
+      <h3 class="p-card__title">In the cloud</h3>
+      <h4>Public and hosted by Canonical</h4>
+      <hr class="u-sv1">
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Periodic automatic updates</li>
+        <li class="p-list__item is-ticked">Secured user and device authentication</li>
+        <li class="p-list__item is-ticked">SLA guarantees</li>
+      </ul>
+      <a class="p-button--positive u-no-margin--bottom" href="https://docs.ubuntu.com/core/en/build-store/">How to create an app store</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: This should be an external button.

File: templates/internet-of-things/appstore.html [hunk 2]
```diff
@@ -195,52 +90,209 @@ <h2 class="p-muted-heading u-align--center">Used by</h2>
...
+      <h3 class="p-card__title">Snap Store Proxy</h3>
+      <hr class="u-sv1">
+      <p class="p-card__content">A proxy gives you all the benefits of an IoT app store with device, OS and application updates all within your firewall.</p>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Pin to a specific version of software on your devices</li>
         <li class="p-list__item is-ticked">Determine which devices connect to your proxy</li>
         <li class="p-list__item is-ticked">Manage users allowed to override revisions</li>
         <li class="p-list__item is-ticked">Maximise bandwidth and speed by caching downloads</li>
       </ul>
+      <a class="p-button--positive u-no-margin--bottom" href="https://docs.ubuntu.com/snap-store-proxy/en/">Snap store proxy</a>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: External button

File: templates/internet-of-things/appstore.html [hunk 3]
```diff
@@ -195,52 +90,209 @@ <h2 class="p-muted-heading u-align--center">Used by</h2>
...
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Enabling IoT ecosystems</h2>
+    </div>
+  </div>
+  <div class="row p-divider u-sv3">
+    <div class="col-4 p-divider__block">
+      <h2>OEMs</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: These should be h3's

File: templates/internet-of-things/appstore.html [hunk 4]
```diff
@@ -195,52 +90,209 @@ <h2 class="p-muted-heading u-align--center">Used by</h2>
...
+      <p>Build new business models around your devices. Seize the opportunity to create platform business models based on software-defined devices.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2>Software vendors</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: h3

File: templates/internet-of-things/appstore.html [hunk 5]
```diff
@@ -195,52 +90,209 @@ <h2 class="p-muted-heading u-align--center">Used by</h2>
...
+      <p>Create a store to publish your software or re-sell software from other vendors. New distribution channels and brand visibility.</p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h2>System integrators</h2>
```
Comment (anthonydillon) [inline templates/internet-of-things/appstore.html]: h3

**Comment by anasereijo** [comment]
- [ ] hero section: make text `7col` and move 'for IoT' to the 2nd line 

- [ ]  Ready for business: make this section `is-default` instead of deep

- [ ] Rigado section:  make this section `is-default` instead of deep

- [ ] Enabling IoT ecosystems: make subtitles 'h4'

- [ ] What use cases?: intro copy should be `h4` as per copydoc

- [ ] What about hosting?:  intro copy should be `h4` as per copydoc

### mobile

- [ ] can we remove the extra bottom padding 

Screenshot 2019-10-28 at 14 45 46

- [ ] fix the spacing between list items

Screenshot 2019-10-28 at 14 46 37

**Comment by anasereijo** [comment]
- [ ] hero make illustration 5 col. It's looking like this:

Screenshot 2019-10-28 at 15 05 24

- [ ]  Ready for business: make this section is-default instead of deep -- Looks the same

- [ ] "What use cases?" As discussed please leave the matrix titles as h4

- [ ] "How to securely manage your IoT fleet" As discussed please leave the matrix titles as h4

**Comment by sowasred2012** [comment]
@pmahnke I've updated this with the latest copy doc changes. The "We would love to hear from you!" section in the copy doc suggests they want a contact form there, so wanted to confirm we're happy for that to be a "Contact us" button?

File: templates/internet-of-things/appstore.html [hunk 6]
```diff
@@ -10,237 +10,302 @@
...
+<section class="p-strip--light">
   <div class="row u-equal-height">
-    <div class="col-8">
-      <p class="p-heading--three">If you’re looking for ways to distribute and manage software on your IoT devices, sign up for a free consultation.</p>
+    <div class="col-7">
+      <h2>Ready to innovate? <br class="u-hide--small" />We would love to hear from you!</h2>
+      <p class="p-heading--four">Our store team would be happy to hear more about your use cases and share advice. Do not hesitate to contact us.</p>
       <p style="margin-top: 2rem;">
-        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us', 'eventLabel' : 'Telcos - Top CTA' });">Contact us</a>
+        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us' });">Contact us</a>
```
Comment (pmahnke) [inline templates/internet-of-things/appstore.html]:
```suggestion
        <a class="p-button--positive js-invoke-modal" href="/internet-of-things/contact-us?product=appstore" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact us' });">Get in touch</a>
```

Comment (anthonydillon) [reply templates/internet-of-things/appstore.html]: This looks good but I do not have permission to push to this branch. Help to land the fix in a follow up?

**Pull request merged**